python/lsst/ctrl/mpexec/util.py

```diff
@@ -28,90 +28,23 @@
 """Few utility methods used by the rest of a package.
 """
 
-__all__ = ["profile", "printTable", "filterTasks", "subTaskIter"]
+__all__ = ["printTable", "filterTasks", "subTaskIter"]
 
-# -------------------------------
-#  Imports of standard modules --
-# -------------------------------
-import contextlib
-import logging
 from collections.abc import Iterator
 
-# -----------------------------
-#  Imports for other modules --
-# -----------------------------
 import lsst.pex.config as pexConfig
 from lsst.pipe.base import Pipeline, TaskDef
 
-# ----------------------------------
-#  Local non-exported definitions --
-# ----------------------------------
-
-# ------------------------
-#  Exported definitions --
-# ------------------------
-
-
-@contextlib.contextmanager
-def profile(filename: str, log: logging.Logger | None = None) -> Iterator:
-    """Context manager for profiling with cProfile
-
```

Comment:
Just want to check and confirm that this is fully private and safe to remove

Reply:
It is a helper function for the `--profile` option and is identical to the version in `lsst.utils`. I removed the code duplication and the function (and `lsst.ctrl.mpexec.util`) was never meant to be public.